### PR TITLE
Adding Elements Markdown and File Structure

### LINKED ISSUE #188: Content & design for Elements page
This page lives here: /UIpatterns/patterns/elements

- [x] Write intro paragraph
- [x] Design page
- [ ] Format for production

### PR DESCRIPTION
**Pull request by ryanbrownhill**
Resolves #188 

**CHANGELOG**
:memo: Adding elements markdown file
Adding folder structure

### CODE REVIEW

File: language/ui-patterns/patterns/elements/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+type: ??
```
Comment (ryanbrownhill) [inline language/ui-patterns/patterns/elements/index.md]: @Snugug I don't know what content type to add to this. Its doesn't quite fit under "Guidelines" or "Patterns" What do you suggest?

Comment (Snugug) [inline language/ui-patterns/patterns/elements/index.md]: It's neither, really. These are special pages like our home page and 404 page

**Comment by Snugug** [comment]
Where'd that folder emoji come from? If it's just adding stuff, no need for the Emoji.

Please also be cognizant of our [commit message language style guide](https://github.com/IBM-Watson/design-library/blob/develop/CONTRIBUTING.md#git-commit-messages). Present tense, imperative move (`Add` instead of `Adding`)

File: language/ui-patterns/patterns/elements/index.md (hunk 1 above)
Comment (bobbinrobyn) [inline language/ui-patterns/patterns/elements/index.md]: Does it matter that we'll have 3 of the same type of page? The Elements, Components, and Layouts pages will all be this same type of list page.

**Comment by ryanbrownhill** [comment]
Updated Ready for :+1: .

**Comment by bobbinrobyn** [comment]
:+1:

**Comment by britanyponvelle** [comment]
:+1:

**Comment by ryanbrownhill** [comment]
@Snugug :+1: ?

**Comment by poofichu** [comment]
Bueno! :+1:

**Comment by poofichu** [comment]
resolves #188

**Pull request merged**